astra-sim/system/Sys.cc

```diff
@@ -534,32 +534,32 @@ void Sys::register_event(
     Callable* callable,
     EventType event,
     CallData* callData,
-    Tick cycles) {
-  try_register_event(callable, event, callData, cycles);
+    Tick delta_cycles) {
+  try_register_event(callable, event, callData, delta_cycles);
 }
 
 void Sys::try_register_event(
     Callable* callable,
     EventType event,
     CallData* callData,
-    Tick& cycles) {
+    Tick& delta_cycles) {
   bool should_schedule = false;
-  if (event_queue.find(Sys::boostedTick() + cycles) == event_queue.end()) {
+  if (event_queue.find(Sys::boostedTick() + delta_cycles) == event_queue.end()) {
```

Comment:
I can see `Sys::boostedTick() + delta_cycles` are being repeated three times, maybe we can instead use `auto event_time = Sys::boostedTick() + delta_cycles` and use that instead for better readability.